tests/lib/reset.sh

```diff
@@ -44,7 +59,11 @@ reset_classic() {
         systemctl start snapd.socket
 
         # wait for snapd listening
-        while ! printf "GET / HTTP/1.0\r\n\r\n" | nc -U -q 1 /run/snapd.socket; do sleep 0.5; done
+        EXTRA_NC_ARGS="-q 1"
+        if [[ "$SPREAD_SYSTEM" = fedora-* ]]; then
+            EXTRA_NC_ARGS=""
```

Comment:
The -q 1 was already in use even without Fedora before. It's strange that by adding Fedora we're adding custom code that adds the flag just inside Fedora, and disables it elsewhere where it was already working. Seems like the new logic is unnecessary?